### ISSUE TITLE
1.0.6 errors

### ISSUE THREAD

**Issue by jamesplease** [fortunejs/fortune-http#7]
v1.0.6 will not initialize. The error is:

```
TypeError: Invalid value used as weak map key
    at WeakMap.set (native)
    at new WeakMap (native)
    at module.exports (/Users/jmeas/webdev/moolah/node_modules/fortune-http/lib/status_map.js:4:10)
    at createListener (/Users/jmeas/webdev/moolah/node_modules/fortune-http/lib/index.js:108:15)
    at module.exports (/Users/jmeas/webdev/moolah/server/app.js:211:20)
    at throng (/Users/jmeas/webdev/moolah/node_modules/throng/lib/throng.js:25:12)
    at Object.<anonymous> (/Users/jmeas/webdev/moolah/server/index.js:10:1)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
```

Locking the dep to 1.0.5 is a temporary solution. I took a look at the source, but nothing immediately stood out as the issue.

**Comment by gr0uch** [comment]
Yep, I'm able to reproduce this issue, I should fix `status_map.js` to be safer.

**Comment by gr0uch** [comment]
It's fixed and published as `1.0.7`.

**Comment by jamesplease** [comment]
Thanks @daliwali !
